<launch>
  
   <!--Node to convert the laser scan to point cloud-->
  <node type="laser2pointcloud2" pkg="laser_to_pcl"  name="laser2pointcloud2" output="screen">  
    <param name="target_frame" value="laser"/>
    <param name="laser_topic" value="/ld_lrs3611/scan"/>
    <param name="pointcloud_topic" value="/ld_lrs3611/scan_pc"/>
  </node>


    <!-- <node pkg="pointcloud_to_laserscan" type="pointcloud_to_laserscan_node" name="pointcloud_to_laserscan" output="screen">
      <remap from="cloud_in" to="/ld_lrs3611/scan_filtered_pc"/>
      <remap from="scan" to="/ld_lrs3611/scan_filtered_world" />
        <rosparam> -->
            <!-- target_frame: world # Leave disabled to output scan in pointcloud frame -->
            <!-- transform_tolerance: 0.01 -->
            <!-- min_height: 0.0
            max_height: 1.0 -->

            <!-- angle_min: -1.5708 
            angle_max: 4.71238899230957  
            angle_increment:  0.004363323096185923 
            scan_time: 0.125
            range_min: 2
            range_max: 250.0 -->
            <!-- use_inf: false -->
            <!-- inf_epsilon: 1.0 -->

            <!-- # Concurrency level, affects number of pointclouds queued for processing and number of threads used
            # 0 : Detect number of cores
            # 1 : Single threaded
            # 2->inf : Parallelism level -->
            <!-- concurrency_level: 2
        </rosparam>
     </node> -->
</launch>
